packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-accessors--to-es2015/private/output.js

```diff
@@ -1,4 +1,5 @@
-var _initProto, _init_a, _get_a, _set_a, _init_b, _get_b, _set_b, _Foo;
+var _Foo;
```

Comment:
This binding is generated by the class properties transform, so it is still `var`.